compose/config/validation.py

```diff
@@ -121,15 +122,16 @@ def anglicize_validator(validator):
     return 'a ' + validator
 
 
+def _parse_key_from_error_msg(error):
```

Comment:
Sorry for that, at some point I had a reason to move it out but it's no longer valid